commands/ls.go

```diff
@@ -342,10 +342,10 @@ func isActive(h *host.Host) (bool, error) {
 
 	dockerHost := os.Getenv("DOCKER_HOST")
 
-	notStopped := currentState != state.Stopped
+	notStoppedSaved := currentState != state.Stopped && currentState != state.Saved
```

Comment:
Sorry to think of this so late in the game, but perhaps we could just change this to something like:

```go
running := currentState == state.Running
```

The reason for this is that there may be other "invalid" states in the future, but `state.Running` is likely to stay the only valid one.

@dmp42 Any opinions?